specification/mixedreality/resource-manager/Microsoft.MixedReality/preview/2019-02-28-preview/mixedreality.json

```
@@ -0,0 +1,724 @@
+        }
+      }
+    },
+    "SpatialAnchorsAccount" : {
+      "allOf" : [{ "$ref" : "../../../../../common-types/resource-management/v1/types.json#/definitions/TrackedResource"}],
+      "description": "SpatialAnchorsAccount Response.",
+      "type": "object",
+      "properties" :{     
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Property bag.",
+          "$ref": "#/definitions/SpatialAnchorsAccountProperties"
+        }
+      }
+    },
+    "SpatialAnchorsAccountKeys" : {
+      "description": "Spatial Anchors Account Keys",
+      "type": "object",
+      "properties": {
+        "primaryKey" : {
+          "description": "value of primary key.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "secondaryKey" : {
+          "description": "value of secondary key.",
+          "readOnly": true,
+          "type": "string"
+        }
+      } 
+    },
+    "SpatialAnchorsAccountKeyRegenerateRequest" : {
+      "description": "Spatial Anchors Account Regenerate Key",
+      "type": "object",
+      "properties": {
+        "serial" : {
+          "type" : "integer",
+          "enum": [1, 2],
+          "default" : 1,
+          "x-ms-enum" : {
+            "name" : "Serial",
+            "values" : [
+              {"value": 1, "description": "The Primary Key", "name": "Primary"},
+              {"value": 2, "description": "The Secondary Key", "name": "Secondary"}
+            ]
+          },
+          "description": "serial of key to be regenerated"
+        }
+      } 
+    },
+    "SpatialAnchorsAccountProperties": {
+      "description": "Spatial Anchors Account Customize Properties",
+      "type": "object",
+      "properties": {
+        "accountId": {
+          "description": "unique id of certain Spatial Anchors Account data contract.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "accountDomain": {
+          "description": "Correspond domain name of certain Spatial Anchors Account",
+          "readOnly": true,
+          "type": "string"
+        }
+      }
+    },
+    "SpatialAnchorsAccountList": {
+      "description": "Result of the request to get resource collection. It contains a list of resources and a URL link to get the next set of results.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SpatialAnchorsAccount"
+          },
+          "description": "List of resources supported by the Resource Provider."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "URL to get the next set of resource list results if there are any."
+        }
+      }
+    }
+  },
+  "parameters": {
+    "apiVersionParameter": {
+      "name": "api-version",
+      "description": "Version of the API to be used with the client request. The current version is 2018-08-08.",
```

You may not want to include an API Version in this description. It will be difficult to remember to update.
```suggestion
      "description": "Version of the API to be used with the client request.",
```